packages/shared/src/components/Feed.tsx

```diff
@@ -330,11 +263,11 @@ export default function Feed<T>({
       )}
       style={style}
     >
-      {openedPost && (
+      {article && (
         <PostModal
-          id={openedPost}
-          isOpen={!!openedPost}
-          onRequestClose={() => setOpenedPost('')}
+          isOpen
+          id={article.id}
+          onRequestClose={onCloseModal}
           navigation={{ onPreviousPost: onPrevious, onNextPost: onNext }}
```

Comment:
It'd better to split this prop to `onPreviousPost` and `onNextPost` rather than a single prop

Reply:
This component actually extends prop from `ModalProps` but just wanted to isolate the props for navigation. Though we can definitely separate it. Let me refactor it quickly! 🚢